### DISCUSSION TITLE
several pixels were inverted in UVT

### DISCUSSION THREAD

**Discussion by forvad999** [sn4k3/UVtools#917] [Q&A]
Hi everyone!
Here's the deal: when opening a file sliced in ChituBox, several pixels were inverted in UVT (100% white to 100% black and vice versa) - circled in red in the picture. On the left - ChituBox, on the right - UVT. Why and how to fix it?
<img width="518" alt="321568 copy" src="https://github.com/user-attachments/assets/6283b6dd-6a55-4a23-8759-ad4f6bc52720">

**Comment by sn4k3** [comment]
That's strange, looks like only affecting small circle with a 1px center.
Can you share the stl and ctb?
Did you also tried with AA off to see the result?

**Comment by forvad999** [comment]
What is AA?
If I move the model, those white/black pixels in the center appear elsewhere.
Here are links to stl and ctb: https://drive.google.com/file/d/1Azfb2iBdJUKcWUCqj4vezhpQhIyVykCM/view?usp=sharing https://drive.google.com/file/d/12hDTw_RyhYwafDx5Qd0zyRMkYtHvB6jR/view?usp=sharing

**Reply by sn4k3** [reply]
AA = AntiAnalising

**Reply by sn4k3** [reply]
Next step is to dry print with a sheet of paper on lcd. I also recommend to cut the STL at the problem and create just a few layers. With your phone video or zoom on that detail and check which preview is correct. Only that way we can know at which side problem is.
If at printer prints is equal to UVtools preview then is chitubox preview which is wrong, otherwise is UVtools.
Please confirm that and report back.

**Comment by forvad999** [comment]
Ok, I get it, will do a bit later - printer is busy printing right now.

**Comment by forvad999** [comment]
![photo_2024-08-28_19-59-15](https://github.com/user-attachments/assets/3a5f41af-e2a0-49da-9f43-537516657c03)
Hi, not enough camera resolution, but I think the third row doesn't have the black pixels in the center like in the UVT preview.

**Reply by sn4k3** [reply]
You need a better picture, compression of low res cameras can smooth pixels and even with zoom we can't tell for sure.
Don't you have a phone with more capable camera?
